packages/manager/modules/manager-pci-common/package.json

```diff
@@ -1,54 +1,79 @@
 {
   "name": "@ovh-ux/manager-pci-common",
   "version": "0.11.0",
-  "private": true,
-  "sideEffects": true,
+  "private": false,
+  "description": "Public Cloud Common components",
+  "homepage": "https://github.com/ovh/manager/blob/master/packages/manager/modules/manager-pci-common/README.md",
+  "bugs": {
+    "url": "https://github.com/ovh/manager/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/manager-pci-common"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
   "type": "module",
-  "main": "./src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "build": "tsc --project tsconfig.build.json",
-    "dev": "tsc",
+    "build": "vite build",
+    "clean": "rimraf node_modules && rimraf dist && yarn cache clean",
+    "dev": "vite build",
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
+    "prepare": "vite build",
+    "prettier": "prettier --write \"src/**/*.{ts,tsx,js,mdx}\"",
     "test": "vitest run",
     "test:coverage": "vitest run --coverage",
-    "test:coverage:watch": "vitest --coverage",
-    "test:watch": "vitest"
+    "test:watch": "vitest --watch",
+    "type:check": "tsc --noEmit"
   },
   "dependencies": {
+    "@ovh-ux/manager-tailwind-config": "^0.2.1",
     "clsx": "2.1.1",
     "date-fns": "^3.6.0",
-    "element-internals-polyfill": "^1.3.11",
-    "file-saver": "^2.0.5",
-    "i18next": "^23.8.2",
-    "i18next-http-backend": "^2.6.1",
     "lodash.isequal": "^4.5.0"
   },
   "devDependencies": {
-    "@jest/globals": "^29.7.0",
     "@ovh-ux/manager-core-api": "^0.9.0",
-    "@ovh-ux/manager-react-components": "^1.41.1",
+    "@ovh-ux/manager-react-components": "^1.43.0",
     "@ovh-ux/manager-react-shell-client": "^0.8.2",
     "@ovh-ux/manager-vite-config": "^0.8.3",
+    "@ovhcloud/ods-common-core": "^17.2.2",
+    "@ovhcloud/ods-common-stencil": "^17.2.2",
+    "@ovhcloud/ods-common-theming": "^17.2.2",
+    "@ovhcloud/ods-components": "^17.2.2",
+    "@ovhcloud/ods-theme-blue-jeans": "^17.2.2",
+    "@tanstack/react-query": "^5.51.21",
```

Comment:
I feel like these dependencies should be in classic dependencies or peer dependencies

Reply:
@SimonChaumet 
They are both in `devDependencies` and `peerDependencies`:

1. in `peerDependencies` to ensure that the client install these packages because they are mandatory at runtime. But, we do not want to bundle them through `dependencies` to prevent for maximizing the bundle size
2. in `devDependencies` because they are needed to develop the library.